src/lms-harmonizer/tests_integration_sql/helpers/sql_scripts.py

```diff
@@ -0,0 +1,212 @@
+        [EducationOrganizationId] ASC,
+        [StudentIdentificationSystemDescriptorId] ASC,
+        [StudentUSI] ASC
+    ) WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
+) ON [PRIMARY];
+END;
+        """
+
+
+CREATE_TABLE_LMS_SECTION = """
+CREATE TABLE lms.LMSSection (
+    LMSSectionIdentifier INT NOT NULL IDENTITY,
+    SourceSystemIdentifier NVARCHAR(255) NOT NULL,
+    SourceSystem NVARCHAR(255) NOT NULL,
+    SISSectionIdentifier NVARCHAR(255) NULL,
+    Title NVARCHAR(255) NOT NULL,
+    SectionDescription NVARCHAR(1024) NULL,
+    Term NVARCHAR(60) NULL,
+    LMSSectionStatus NVARCHAR(60) NULL,
+    SourceCreateDate DATETIME2(7) NULL,
+    SourceLastModifiedDate DATETIME2(7) NULL,
+    EdFiSectionId UNIQUEIDENTIFIER NULL,
+    CreateDate DATETIME2 NOT NULL,
+    LastModifiedDate DATETIME2 NOT NULL,
+    DeletedAt DATETIME2(7) NULL,
+    CONSTRAINT LMSSection_PK PRIMARY KEY CLUSTERED (
+        LMSSectionIdentifier ASC
+    ) WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
+) ON [PRIMARY];
+
+ALTER TABLE lms.LMSSection ADD CONSTRAINT LMSSection_DF_CreateDate DEFAULT (getdate()) FOR CreateDate;
+ALTER TABLE lms.LMSSection ADD CONSTRAINT LMSSection_DF_LastModifiedDate DEFAULT (getdate()) FOR LastModifiedDate;
+ALTER TABLE lms.LMSSection ADD CONSTRAINT LMSSection_UK_SourceSystem UNIQUE (SourceSystemIdentifier, SourceSystem);
+"""
+
+
+CREATE_TABLE_EDFI_SECTION = """
+CREATE TABLE [edfi].[Section](
+    [LocalCourseCode] [nvarchar](60) NOT NULL,
+    [SchoolId] [int] NOT NULL,
+    [SchoolYear] [smallint] NOT NULL,
+    [SectionIdentifier] [nvarchar](255) NOT NULL,
+    [SessionName] [nvarchar](60) NOT NULL,
+    [SequenceOfCourse] [int] NULL,
+    [EducationalEnvironmentDescriptorId] [int] NULL,
+    [MediumOfInstructionDescriptorId] [int] NULL,
+    [PopulationServedDescriptorId] [int] NULL,
+    [AvailableCredits] [decimal](9, 3) NULL,
+    [AvailableCreditTypeDescriptorId] [int] NULL,
+    [AvailableCreditConversion] [decimal](9, 2) NULL,
+    [InstructionLanguageDescriptorId] [int] NULL,
+    [LocationSchoolId] [int] NULL,
+    [LocationClassroomIdentificationCode] [nvarchar](60) NULL,
+    [OfficialAttendancePeriod] [bit] NULL,
+    [SectionName] [nvarchar](100) NULL,
+    [Discriminator] [nvarchar](128) NULL,
+    [CreateDate] [DATETIME2](7) NOT NULL,
+    [LastModifiedDate] [DATETIME2](7) NOT NULL,
+    [Id] [uniqueidentifier] NOT NULL,
+    [ChangeVersion] [bigint] NOT NULL,
+ CONSTRAINT [Section_PK] PRIMARY KEY CLUSTERED
+(
+    [LocalCourseCode] ASC,
+    [SchoolId] ASC,
+    [SchoolYear] ASC,
+    [SectionIdentifier] ASC,
+    [SessionName] ASC
+)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON, OPTIMIZE_FOR_SEQUENTIAL_KEY = OFF) ON [PRIMARY]
```

Comment:
```suggestion
)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
```

Removing something that was introduced in SQL Server 2019

Reply:
✔️  removed